astroplan/plots/core.py

```diff
@@ -0,0 +1,223 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+import random
+
+
+def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
+                 dark_plot=False):
+    """
+    TODO:
```

Comment:
I don't mind having a TODO in the docstring (others might disagree..?) but please put it below the summary and description (just before `Parameters`)